### ISSUE TITLE
Some item imports from Simc Addon not scaling correctly in simc

### ISSUE THREAD

**Issue by Niphyr** [simulationcraft/simc#5490]
### Describe the bug
Item scaling for some items on Beta isn't working as expected when using the `/simc` addon input.
```
0.000 Player Niphcovtest item 'lingering_sunmote' adjusting quality (old=rare new=epic)
0.000 Niphcovtest: Scaling lingering_sunmote from ilevel 158 to 226 (226.000) using player level 50
0.000 Player Niphcovtest item 'lingering_sunmote' adjusting ilevel by 75 (old=226 new=301)
0.000 Player Niphcovtest item 'lingering_sunmote' adjusting quality (old=epic new=epic)
0.000 Player Niphcovtest item 'unbound_changeling' adding effect 330740 (type=equip, index=1)
0.000 Player Niphcovtest item 'unbound_changeling' adjusting quality (old=rare new=epic)
0.000 Niphcovtest: Scaling unbound_changeling from ilevel 158 to 226 (226.000) using player level 50
0.000 Player Niphcovtest item 'unbound_changeling' adjusting ilevel by 75 (old=226 new=301)
0.000 Player Niphcovtest item 'unbound_changeling' adjusting quality (old=epic new=epic)
```
So far only experiencing this with trinkets.

### Expected behavior
Expecting the items imported via `/simc` to be scaled to the same ilvl as in-game
![image](https://user-images.githubusercontent.com/1481378/97963544-73ba6100-1e0b-11eb-8fc5-26f28c37959a.png)

### To Reproduce
1. From the trinket vendor on beta purchase a trinket such as Unbound Changeling (item id 178708)
2. Using SimC Addon 9.0.1-02 open `/simc` and copy the import line for that trinket
```
trinket1=,id=178708,bonus_id=6652/6917/7215/1547/6646,drop_level=50
```
3. Use that as part of a profile and run the sim
### Additional information
* SimulationCraft Version (required): Latest from github built locally in VS as of the time of this report

**Comment by scamille** [comment]
cc @navv1234 @seriallos

**Comment by navv1234** [comment]
The problem here is that in game it seems the item gets two item bonuses that apply ilevel adjustments to the item. One that is based on a curve and drop level of the item, and another that simply performs a static ilevel adjustment on top of the static base ilevel.

Simc itself is simply applying both to the item, resulting in inflated ilevels. We will need to figure a way to work around this somehow.

**Comment by navv1234** [comment]
Commit eccd619 implements a wild guess at what is going on in game, but honestly, kinda impossible to know. Ilevels probably need to be relatively closely monitored when we get "natural" item drops instead of trinket vendor to see if there's something weird going on.

**Comment by scamille** [comment]
Can this be closed? If I read this correctly, this affected only items from a special beta vendor which blizzard somehow manually forced to a specific item level.

**Comment by navv1234** [comment]
Maybe? The code is still in, and we're not sure if those item bonuses are in the wild. Regardles, if they crop up someone will reopen an issue about it I guess.

**Comment by scamille** [comment]
Let me rephrase it: I assume that there currently aren't any widely used items using those problematic multiple item bonuses affecting item level in the live version of the game, so we haven't yet seen any item level discrepancies. I also assume that if there were problems in the items used in the PreRaid or T26 profiles, we would have heard about it by now.

Anyway, closing this then.
